pkg/cmd/artefactor.go

```diff
@@ -1,17 +1,29 @@
 package cmd
 
 import (
+	"fmt"
+	"io/ioutil"
+	"log"
 	"os"
 	"strings"
 
 	"github.com/spf13/cobra"
 )
 
 const (
+	EnvPrefix = "ARTEFACTOR_"
 	// FlagDockerImages specifies a to newline seperated list of docker images to save
 	FlagDockerImages = "docker-images"
-	// The directory to save archives into
+	// FlagArchiveDir is the directory to save archives into
 	FlagArchiveDir = "archive-dir"
+	// FlagGitRepos specifies a newline speerated list of local or remote git repos
+	FlagGitRepos = "git-repos"
+	// FlagWebFiles specifies a whitespace delimited set of csv's with url,file,md5
```

Comment:
With your example in the README there's also a bool on the end?

Reply:
I've improved the readme and made the comment accurate.